superset-frontend/src/SqlLab/components/ExploreCtasResultsButton/index.jsx

```diff
@@ -68,39 +62,35 @@ class ExploreCtasResultsButton extends React.PureComponent {
           all_columns: [],
           row_limit: 1000,
         };
-        this.props.actions.addInfoToast(
-          t('Creating a data source and creating a new tab'),
-        );
+        addInfoToast(t('Creating a data source and creating a new tab'));
 
         // open new window for data visualization
         exploreChart(formData);
       })
       .catch(() => {
-        this.props.actions.addDangerToast(
-          this.props.errorMessage || t('An error occurred'),
-        );
+        addDangerToast(errorMessage || t('An error occurred'));
       });
-  }
+  };
 
-  render() {
-    return (
-      <>
-        <Button
-          buttonSize="small"
-          onClick={this.onClick}
-          tooltip={t('Explore the result set in the data exploration view')}
-        >
-          <InfoTooltipWithTrigger
-            icon="line-chart"
-            placement="top"
-            label="explore"
-          />{' '}
-          {t('Explore')}
-        </Button>
-      </>
-    );
-  }
-}
+  const onClick = () => {
+    visualize();
+  };
+
+  return (
+    <Button
+      buttonSize="small"
+      onClick={onClick}
+      tooltip={t('Explore the result set in the data exploration view')}
+    >
+      <InfoTooltipWithTrigger
+        icon="line-chart"
+        placement="top"
+        label="explore"
+      />{' '}
+      {t('Explore')}
+    </Button>
+  );
+};
 ExploreCtasResultsButton.propTypes = propTypes;
 
 function mapStateToProps({ sqlLab, common }) {
```

Comment:
Please change the `mapStateToProps` and `mapDispatchToProps` functions to arrow functions.
```suggestion
const mapStateToProps = ({ sqlLab, common }) => ({
  errorMessage: sqlLab.errorMessage,
  timeout: common.conf ? common.conf.SUPERSET_WEBSERVER_TIMEOUT : null,
});

const mapDispatchToProps = dispatch => ({
  actions: bindActionCreators(actions, dispatch),
});
```